src/main/java/org/roda_project/commons_ip2/utils/METSUtils.java

```diff
@@ -59,7 +61,7 @@ public static Mets instantiateMETSFromFile(Path metsFile) throws JAXBException,
     Source metsSchemaSource = new StreamSource(metsSchemaInputStream);
     Schema schema = factory.newSchema(metsSchemaSource);
     jaxbUnmarshaller.setSchema(schema);
-    return (Mets) jaxbUnmarshaller.unmarshal(metsFile.toFile());
+    return (Mets) jaxbUnmarshaller.unmarshal(new BufferedInputStream(Files.newInputStream(Paths.get(metsFile.toString()))));
```

Comment:
metsFile is already a Path, there is no need to use Paths.get method
```suggestion
    return (Mets) jaxbUnmarshaller.unmarshal(new BufferedInputStream(Files.newInputStream(metsFile.toAbsolutePath())));
```